examples/CivisML_parallel_training.ipynb

```diff
@@ -153,7 +153,7 @@
     "\n",
     "print(\"Shape: \", df.shape)\n",
     "\n",
-    "df['is_cancer'] = [1 if val == 4 else 0 for val in df['is_cancer'] ]\n",
+    "df['is_cancer'] = [1 if val == 4 else (0 if val == 2 else '?') for val in df['is_cancer'] ]\n",
```

Comment:
I originally left it off to keep it closer to what the author used. Though I agree np.nan is better. I added it (along with changelog modification). For clarity I added an import to numpy rather than use it via pandas since pd.np.nan can be confusing.